stl/src/vector_algorithms.cpp

```diff
@@ -78,6 +81,106 @@ namespace {
 
 extern "C" {
 
+#ifdef _M_ARM64
+__declspec(noalias) void __cdecl __std_swap_ranges_trivially_swappable_noalias(
+    void* _First1, void* const _Last1, void* _First2) noexcept {
+    constexpr size_t _Mask_64 = ~((static_cast<size_t>(1) << 6) - 1);
+    if (_Byte_length(_First1, _Last1) >= 64) {
+        const void* _Stop_at = _First1;
+        _Advance_bytes(_Stop_at, _Byte_length(_First1, _Last1) & _Mask_64);

```

Comment:
`_Mask_64` can be scoped more tightly. (I'll apply this to the existing x64/x86 code in a followup PR to avoid distracting changes here.)